src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/InertContentType.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
+{
+    // It's possible to get this via ITextBufferFactoryService.InertContentType,
+    // but plumbing it through is ugly and this can be used in unit tests as well
+    internal class InertContentType : IContentType
+    {
+        public static readonly IContentType Instance = new InertContentType();
+
+        public string TypeName => "inert";
+
+        public string DisplayName => TypeName;
+
+        public IEnumerable<IContentType> BaseTypes => Enumerable.Empty<IContentType>();
+
+        public bool IsOfType(string type)
+        {
+            if (string.Equals(type, TypeName, StringComparison.OrdinalIgnoreCase))
+            {
+                return true;
+            }
+
+            return false;
+        }
```

Comment:
```suggestion
        public bool IsOfType(string type) =>
            string.Equals(type, TypeName, StringComparison.OrdinalIgnoreCase);
```

Reply:
Yep, agreed.

---
In reply to: [451968659](https://github.com/dotnet/aspnetcore-tooling/pull/2201#discussion_r451968659) [](ancestors = 451968659)